spec/models/exporters/csv_subject_reduction_exporter_spec.rb

```diff
@@ -1,94 +0,0 @@
-require 'spec_helper'
-
-describe Exporters::CsvSubjectReductionExporter do
```

Comment:
Where did these tests end up post-refactor? I see that in the request worker spec it builds the files, but nothing tests the contents except this deleted block.

Reply:
hm, that's probably true, they should have been picked up again somewhere